src/services/Challenge/ChallengeGlobal/ChallengeGlobal.js

```diff
@@ -0,0 +1,7 @@
+export const challengePassesGlobalFilter = function(filter, challenge) {
+  if (filter.global) {
+    return challenge.isGlobal === false
+  }
```

Comment:
Is this condition backwards? It's written differently from `challengePassesArchivedFilter` even though the naming of the `showingGlobal` and `showingArchived` booleans implies that they'd work the same (true means show challenges matching the criteria, false means hide them).

Reply:
Thanks for catching that, you are right. Just updated it.